telemetry/aiwrapper.go

```diff
@@ -2,16 +2,18 @@
 package telemetry
 
 import (
-	"fmt"
+	"errors"
+
 	"github.com/Azure/azure-container-networking/aitelemetry"
 	"github.com/Azure/azure-container-networking/log"
 )
 
 var (
-	aiMetadata     string
-	th             aitelemetry.TelemetryHandle
-	gDisableTrace  bool
-	gDisableMetric bool
+	aiMetadata           string
+	th                   aitelemetry.TelemetryHandle
+	gDisableTrace        bool
+	gDisableMetric       bool
+	ErrTelemetryDisabled = errors.New("telemetry is disabled")
```

Comment:
The variable name should be `errTelemetryDisabled`

Reply:
probably not, because then I can't compare to it with `errors.Is/As` from outside of this package

Reply:
Oh got it. Thanks @rbtr

Reply:
Reverted back to ErrTelemetryDisabled